src/appleseedmaya/renderglobalsnode.h

```diff
@@ -205,6 +205,8 @@ class RenderGlobalsNode
 
     // Render stamp.
     static MObject      m_renderStamp;
+    static MObject      m_renderStampScaleFactor;
+    static MObject      m_renderStampOrder;
```

Comment:
Did you mean to expose the stamp order?

Reply:
When more post-process effects are added, stamp order will have to be exposed. For now it is not really required and I remove it.

Ps) Thanks for the review!